packages/code-block/partials/code-tabs/index.test.js

```diff
@@ -25,6 +26,35 @@ it('should accept a className prop, and place it on the root element', () => {
   expect(container.firstChild.className).toContain(customClassName)
 })
 
+it('should accept CodeBlockConfig or pre elements as children', () => {
```

Comment:
Added a test to assert that the changes made don't affect the existing behaviour.